conf/messages

```diff
@@ -460,34 +460,29 @@ property.details.cases.table.challenge=Challenge
 property.dvr.challenge.error=If you still think you have the right to make a Challenge, you will need to <a class="govuk-link" href="{0}"> download and complete a Challenge form (T012)</a>
 
 assessments.table.caption=Valuations
-assessments.table.1=Effective date
-assessments.table.2=Rateable value
-assessments.table.3=Capacity
-assessments.table.4=From
-assessments.table.5=To
-assessments.table.6=Actions
-assessments.table.action=Action
+assessments.table.1=Valuation
+assessments.table.2=Effective date
+assessments.table.3=Connection to property
+assessments.table.4=Rateable value
 assessments.enddate.present=Present
 assessments.enddate.present.lowercase=present
 assessments.enddate.historic.empty=31 March 2023
 assessments.view.valuation=View valuation
 assessments.view.summary=View summary valuation
 label.viewAssessmentRadio=Valuation
 
-assessments.localAuthorityRef=Local authority reference:
-assessments.property=Property:
+assessments.localAuthorityRef=Local council reference:
+assessments.caption.ip=Your property
+assessments.caption.agent=Client property: {0}
 assessments.title=Select a valuation to check
-assessments.pageTitle=Select a valuation to check
-assessments.paragraph=The rateable value of a property is reassessed by the VOA each time they publish a new business rates valuation list. You may check the facts and the rateable value for each of the published valuations.
-
-assessments.current.title=Current valuation
-assessments.draft.title=Future valuation
-assessments.historic.title=Previous valuations
-assessments.select.valuation=Select this valuation
+assessments.subtitle=Valuations for this property
 
+assessments.icon.help.valuationPeriod=Help with valuation period
```

Comment:
In the tickets comment it mentions "The english version should read 'Help with valuation' rather than 'Help with valuation period'."

That comment was only added yesterday so this may need updating.

Reply:
If this does need updating to 'Help with valuation' it would be worth checking if we need to request new welsh translation.